submission/distributor.go

```diff
@@ -36,6 +37,11 @@ import (
 	ct "github.com/google/certificate-transparency-go"
 )
 
+var (
+	DistributorNotEnoughCompatibleLogsErr   = errors.New("distributor does not have enough compatible Logs to comply with the policy")
+	DistributorUnableToProcessEmptyChainErr = errors.New("distributor unable to process empty chain")
```

Comment:
For error sentinels, one typically prefixes their name with the word "Err."

Reply:
Ugh I always mistake these. I'll make a followup PR addressing these comments.